src/Read_Forcing.f90

```diff
@@ -63,7 +63,10 @@ SUBROUTINE Read_Forcing
 !
   no_heat=no_heat+1 
   Q_out(no_heat)=Q_out(no_heat-1)
-!  Q_trib(nr)=Q_out(no_heat)    
+!
+! Tributary flow from this reach equals Q_out for this cell
```

Comment:
It took me a few minutes to wrap my head around why Q_out = Q_trib (I kept thinking Q_trib was tributary input into this reach).  Maybe make the comment something like "Q_out at the outlet of this reach is Q_trib, when the flow (Q_trib) at the outlet of this reach enters the downstream reach (i.e. when this reach is a tributary)".